.ci/filter/filter_k8s_test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+GOPATH_LOCAL="${GOPATH%%:*}"
+KATA_DIR="${GOPATH_LOCAL}/src/github.com/kata-containers"
+TEST_DIR="${KATA_DIR}/tests"
+CI_DIR="${TEST_DIR}/.ci"
+
+K8S_FILTER_FLAG="kubernetes"
+
+source "${CI_DIR}/lib.sh"
+
+main()
+{
+	local K8S_CONGIF_FILE="$1"
```

Comment:
Typo: `K8S_CONFIG_FILE` (here and below) :smile:

Reply:
nice catch~😎